src/webview/js/rewriteHtml.js

```diff
@@ -86,6 +86,26 @@ const rewriteTime = (element: Element | Document) => {
   });
 };
 
+/**
+ * Add missing elements to spoiler blocks under the given root, inclusive.
+ *
+ * 1. Adds 'Spoiler' header if missing.
+ * 2. Adds the spoiler expanded arrow indicator. The arrow is rendered using CSS.
+ */
+const rewriteSpoilers = (element: Element | Document) => {
```

Comment:
This should get a comment pointing at the corresponding code in the webapp -- which is in rendered_markdown.js, at the spot you find searching for "spoiler-header".

(I was wondering why it's needed, and whether it corresponds to anything in the webapp. Turns out it does.)